src/cloud-cli/userdb.ts

```diff
@@ -156,18 +156,18 @@ export function migrate(): number {
 
   if (create_db) {
     logger.info(`Creating database ${userdbname}`);
-    const cmd = `createdb -h ${configFile.database.hostname} -p ${configFile.database.port} ${userdbname} -U ${configFile.database.username} -w ${configFile.database.password} -e`;
+    const cmd = `PGPASSWORD=${configFile.database.password} createdb -h ${configFile.database.hostname} -p ${configFile.database.port} ${userdbname} -U ${configFile.database.username} -ew ${userdbname}`;
     logger.info(cmd);
     try {
-      execSync(cmd);
+      const createDBCommandOutput = execSync(cmd).toString();
+      logger.info(createDBCommandOutput);
     } catch (e) {
       logger.error("Database already exists or could not be created.");
-      return 1;
```

Comment:
We need to tolerate this failure

Reply:
If the DB doesn't exist, that isn't even an error